core/src/main/scala/com/databricks/labs/remorph/parsers/intermediate/plans.scala

```diff
@@ -122,6 +122,7 @@ abstract class Plan[PlanType <: Plan[PlanType]] extends TreeNode[PlanType] {
     // Recursively find all expressions from a traversable.
     def seqToExpressions(seq: Iterable[Any]): Iterable[Expression] = seq.flatMap {
       case e: Expression => e :: Nil
+      case p: Plan[_] => p.expressions
```

Comment:
This changes the behavior of the `expressions` method in a way that deserves to be reflected in the documentation. Please change the documentation so that users are aware that this returns all expressions that appear in `this` plan *and all its descendants*.

Reply:
done